hr_recruitment_partner/README.rst

```diff
@@ -0,0 +1,79 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Applicants implicit Partner
+===========================
+
+Automatically create a Partner for Applicants.
+
+When an Applicant is created, a related Partner for it is automatically
+created.
+
+Duplicate personal data is directly stored in the related Partner record,
+but is still available in the Applicant model.
+
+As an additional advantage, it is easier now to add other personal and
+address details to the Applicant form, such as the photo image or the
+address details.
+
+When installing the first time, related Partners will automatically be created
+for existing Applicants not having one.
+
+
+Configuration
+=============
+
+On install, new Partner record are automatically created for existing
```

Comment:
This sentence seems to be redundant. You just wrote the same thing few lines upper.